scripts/poclcc/poclcc.c

```diff
@@ -0,0 +1,179 @@
+#include <stdio.h>
```

Comment:
There is no _bin_ folder at the moment, should I create one? or just let _poclcc.c_ in the script folder and clean every trace of pocl-standalone?